inject.js

```diff
@@ -275,33 +286,75 @@
           // the first element of the target, which may not be the parent.
           this.parent.insertBefore(fragment, this.parent.firstChild);
       }
-	  return wrapper;
+      return wrapper;
     }
   }
 
-  function initializeWhenReady(document) {
```

Comment:
This looks worse than it really is, just pushed out blacklist into its own function and likewise with blacklistrc. initializeWhenReady looks much cleaner now, and it's easy to identify if blacklisted from anywhere now.